### PR TITLE
Add tooltips text

### LINKED ISSUE #533: Improve tooltips and some texts in the app
WIP:
https://docs.google.com/document/d/1p-KUa14y59Ny5BeCD0J0Ft6Nx_Zik0_QGNbDaNQH3Fo/edit

### Estimation:
*2*

### PR DESCRIPTION
**Pull request by Velenir**
Adds tooltips with text as per the doc
Bu I really don't like how long tooltips look
And in general tooltips are commonly short and to the point.

localhost_8080_liquidity (1)

localhost_8080_liquidity (2)

Any ideas what we could do? More text somewhere instead of tooltips? Different styling?

Closes #533

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr731--dexreact.review.gnosisdev.com

File: src/hooks/usePopper.ts [hunk 1]
```diff
@@ -116,10 +116,27 @@ interface PopperDefaultHookResult<T extends HTMLElement> {
...
+    if (offset !== undefined) {
+      config.modifiers = [
+        {
+          name: 'offset',
+          options: {
+            offset: [0, offset],
+          },
+        },
+      ]
+      console.log('config.modifiers', config.modifiers)
```
Comment (alfetopito) [inline src/hooks/usePopper.ts]: remove

**Comment by Rafanator** [comment]
I agree, too much text. Lets change to:

1. This balance reflects the amount deposited in the Exchange Wallet on Mesa, not the overall amounts in your wallet.
2. Your order will be considered for the next upcoming batch.
3. If, for example, you select DAI and USDC with a spread of 1%, everytime the price of DAI is 1% above USDC you might sell your DAI for USDC. If USDC is 1% above DAI (i.e. 1.01), you might sell your USDC for DAI.

Usually Chris makes the final call on this, so let's do the changes and ask her when she is back if she is ok with them.

**Comment by W3stside** [comment]
agree agree

**Comment by anxolin** [comment]
What about the part of adding also this help icons so the user has to click on the help to get it:
1. It's less intrusive
2. It's more mobile friendly. There's no hover events in mobile
3. When a user clicks something to get more info, he is more likely to read the information

**Comment by Velenir** [comment]
> 2\. It's more mobile friendly. There's no hover events in mobile

On mobile hover gets activated onClick

**Comment by anxolin** [comment]
> On mobile hover gets activated onClick

I still thing is not a good UX

**Comment by Velenir** [comment]
> I agree, too much text. Lets change to:

localhost_8080_liquidity (1)

localhost_8080_liquidity

slightly better

**Comment by Velenir** [comment]
> What about the part of adding also this help icons

That sounds better. Like a quiestion mark :question: :grey_question:  that would open a tooltip with a different styling, or even a modal

**Comment by anxolin** [comment]
> That sounds better. Like a quiestion mark ❓ ❔ that would open a tooltip with a different styling, or even a modal

Exactly, that icon, or a (i) as suggested in the early requirement proposals:
image

**Comment by Velenir** [comment]
:question: tooltips added in #746, please take a look

**Comment by Velenir** [comment]
Can I have final confirmation that all is good, especially from @Rafanator and @c3rnst 
And approvals on this PR

**Comment by anxolin** [comment]
I'm merging for doing an user test. Any new change, pls create an issue

**Pull request merged**